### ISSUE TITLE
Burning fees in OEP

### ISSUE THREAD

**Issue by drhashes** [drhashes/Burnfees-OEP#1]
@baginail
@malefol7

After the London fork the open ethereum pool no longer works because it gets the block reward wrong.

It needs to subtract burning of fees from the total block reward. For someone not familiar with OEP and golang 
this can be an almost impossible task to fix.

I can sell consulting work to fix this for you on OEP. If interested you may contact me here on github.
Easiest is probably to reply here.

Best regards. //drhashes

**Comment by malefol7** [comment]
hello, calculating the combustion at block level, took many hours of work, modifying a code written by another person is always a difficult task, I found the solution after 10 hours. Thanks for your availability

**Comment by drhashes** [comment]
Right. Not surprised. You actually sounded like you knew what you were doing.
Good luck with pool!

**Comment by drhashes** [comment]
Btw, let me know if you need help in getting outgoing transactions Type: 2 (EIP-1559) to work in OEP.

**Comment by drhashes** [comment]
@DigiPools
@chenhonghe
@Serpent6877

See top of this thread.

**Comment by Serpent6877** [comment]
@malefol7 Did you find the solution on github or fix it yourself?

Thanks!
